src/app/zap-templates/zcl/data-model/chip/thread-border-router-management-cluster.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <bitmap name="Feature" type="bitmap32">
+    <cluster code="0x0452"/>
+    <field name="PANChange" mask="0x1"/>
+  </bitmap>
+
+  <cluster>
+    <domain>HRAP</domain>
+    <name>Thread Border Router Management</name>
+    <code>0x0452</code>
+    <define>THREAD_BORDER_ROUTER_MANAGEMENT</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <description>Manage the Thread network of Thread Border Router</description>
+
+    <globalAttribute code="0xFFFD" side="either" value="1"/>
+
+    <attribute side="server" code="0x0000" define="BORDER_ROUTER_NAME" type="char_string" length="63">BorderRouterName</attribute>
+
+    <attribute side="server" code="0x0001" define="BORDER_AGENT_ID" type="octet_string" length="16">BorderAgentId</attribute>
```

Comment:
This does not match the spec.

Stopping here.  Please go through the XML carefully and make sure it matches the spec.

Reply:
Changed.